hathor/p2p/netfilter/matches.py

```diff
@@ -83,7 +83,7 @@ def match(self, context: 'NetfilterContext') -> bool:
             return False
 
         try:
-            host = ip_address(context.addr.host)
+            host = ip_address(getattr(context.addr, 'host', ''))
```

Comment:
Maybe `host unknown`?

Reply:
I just noticed that we check for a `hasattr` a few lines above, so the value here doesn't matter at all, I'll just remove this parameter, since its optional.

Reply:
Done.

Reply:
Great! Thanks!